eks-anywhere-common/Addons/Partner/SUSE/NeuVector/neuvector-core.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: neuvector
+  namespace: neuvector
+  labels:
```

Comment:
The annotation is not required here. It is required only in `namespace` resource